projects/uitdatabank/reference/entry.json

```diff
@@ -9496,6 +9496,320 @@
           }
         ]
       }
+    },
+    "/cultuurkuur/regions": {
+      "get": {
+        "summary": "/cultuurkuur/regions",
+        "tags": ["Cultuurkuur"],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "headers": {
+              "content-type": {
+                "schema": {
+                  "type": "string",
+                  "example": "application/json"
+                },
+                "description": "application/json"
+              }
+            },
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "label": {
+                        "type": "string",
+                        "description": "Name region"
```

Comment:
During the meeting with Eva, we discussed that these labels should be translatable

> We will not translate regions and educational levels, but we will make sure the nl field is used

```
label: {
 nl: 'Gewoon basisonderwijs'
},
```